buildspec/linuxIntegrationTests.yml

```diff
@@ -23,11 +23,15 @@ phases:
             # Install .NET Core 2.1
             - apt-get install -y dotnet-sdk-2.1
             # Install other needed dependencies
-            - apt-get install -y python2.7 python-pip python3.6 python3.7 python3.8 python3-pip python3-distutils docker
+            - apt-get install -y jq python2.7 python-pip python3.6 python3.7 python3.8 python3-pip python3-distutils docker
             - apt-get install -y libgtk-3-dev libxss1 xvfb libnss3-dev libasound2 libasound2-plugins libsecret-1-0
             - pip3 install --user aws-sam-cli
             - pip3 install --upgrade awscli
             - pip3 install pylint
+            # login to DockerHub so we don't get throttled
+            - export DOCKER_USERNAME=`echo $DOCKER_HUB_TOKEN | jq -r '.username'`
+            - export DOCKER_PASSWORD=`echo $DOCKER_HUB_TOKEN | jq -r '.password'`
+            - docker login --username $DOCKER_USERNAME --password $DOCKER_PASSWORD || true
```

Comment:
don't need to `export`.

    DOCKER_USERNAME=$(echo $DOCKER_HUB_TOKEN | jq -r '.username') DOCKER_PASSWORD=$(echo $DOCKER_HUB_TOKEN | jq -r '.password') docker login --username $DOCKER_USERNAME --password $DOCKER_PASSWORD || true

Reply:
Sure, changed